app/models/api_token.rb

```diff
@@ -0,0 +1,32 @@
+class APIToken < ApplicationRecord
+  belongs_to :user
+
+  validates :user, presence: true
+  validates :token, presence: { message: I18n.t('activerecord.errors.models.api_token.token.blank') },
```

Comment:
```suggestion
  validates :token, presence: true},
```
Message shouldn't be needed if the message is in the right place in `en.yml`